core/logger/audit/audit_logger.go

```diff
@@ -287,24 +287,20 @@ func (l *AuditLoggerService) Close() error {
 	return nil
 }
 
-func (l *AuditLoggerService) Healthy() error {
-	if !l.enabled {
-		return errors.New("the audit logger is not enabled")
-	}
-
-	if len(l.loggingChannel) == bufferCapacity {
-		return errors.New("buffer is full")
-	}
-
-	return nil
-}
-
 func (l *AuditLoggerService) Name() string {
 	return l.logger.Name()
 }
 
 func (l *AuditLoggerService) HealthReport() map[string]error {
-	return map[string]error{l.logger.Name(): l.Healthy()}
+	var err error
+	if !l.enabled {
+		err = errors.New("the audit logger is not enabled")
+	}
+
+	if len(l.loggingChannel) == bufferCapacity {
```

Comment:
Would the second ever overwrite the first?
```suggestion
	} else if len(l.loggingChannel) == bufferCapacity {
```